### ISSUE TITLE
Add region landmarks to Checkout Process

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13519]
**Describe the bug**

- Screen Reader users must be able to navigate by landmarks, currently, there aren't many nor labelled, therefore relevant sections on the page should be identified with meaningful landmarks names.

**Tell us the version of Spartacus**
- Library version: 4.0.0.

**To Reproduce**
1. Open any Screen Reader (VO, NVDA, JAWS).
2. Bring up the Rotor or Element List Menu.
3. Choose Landmarks view.
4. See absence of relevant landmarks.

**Expected behavior**

- Use only predefined landmarks https://wiki.wdf.sap.corp/wiki/display/spar/Landmarks
- Change from `<section>` to `<nav aria-label="Checkout Progress">`

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus using installation script
More extensive testing (on other screenreaders and devices) will be done with RC.

<img width="1238" alt="Screen Shot 2022-03-14 at 2 13 29 PM" src="https://user-images.githubusercontent.com/45409515/158235136-ed3b1824-f5f0-4b5e-bc61-0536cd5de705.png">

<img width="1175" alt="Screen Shot 2022-03-14 at 2 14 36 PM" src="https://user-images.githubusercontent.com/45409515/158235439-1ab90c77-2304-4bdc-b7f9-27f53ce19549.png">
<img width="1223" alt="Screen Shot 2022-03-14 at 2 15 08 PM" src="https://user-images.githubusercontent.com/45409515/158235448-a38de3da-4cfa-40a7-99a3-b99ef47f5639.png">
<img width="1236" alt="Screen Shot 2022-03-14 at 2 15 27 PM" src="https://user-images.githubusercontent.com/45409515/158235455-e78d8a63-a538-4288-88c2-46963581d162.png">
